### PR TITLE
governance and core-dev

### PR DESCRIPTION
**Pull request by MMathisLab**

### CODE REVIEW

File: docs/MISSION_AND_VALUES.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+a welcoming place for all.
+
+- We are **community-engaged**. We respond to feature requests and proposals on our [issue tracker](https://github.com/DeepLabCut/DeepLabCut/issues).
+
+- We serve **scientific applications** primarily, over “consumer or commercial” pose estimation tools. This often means prioritizing core functionality support, and rejecting implementations of “flashy” features that have little scientific value.
+
+- We are **domain agnostic** within the sciences. Functionality that is highly specific to particular scientific domains belongs in plugins, whereas functionality that cuts across many domains and is likely to be widely used belongs inside DeepLabCut.
+
+- We value **education and documentation**. All functions should have docstrings, preferably with examples, and major functionality should be explained in our [tutorials](https://github.com/napari/napari-tutorials). Core developers can take an active role in finishing documentation examples.
```
Comment (jeylau) [inline docs/MISSION_AND_VALUES.md]: The link should be updated :)

**Comment by jeylau** [comment]
Besides the point above, looks great to me!

File: docs/MISSION_AND_VALUES.md (hunk 1 above)
Comment (MMathisLab) [reply docs/MISSION_AND_VALUES.md]: fixed ;0

**Comment by AlexEMG** [comment]
Perfect.

**Pull request merged**